### ISSUE TITLE
[8.0] share: base.view_users_form "AttributeError: Field `in_group_*` does not exist" after "-i base"

### ISSUE THREAD

**Issue by moylop260** [odoo/odoo#7885]
Steps to reproduce:
 1. create a new database: e.g. `createdb share_bug`
 2. Start odoo installing share module: e.g. `odoo.py -d share_bug -i share`
 3. Go to res.users form view. Result: All fine!
 4. Restart odoo with `-u base` and later with `-i base`
 5. Go to res.users form view again. Result: Error!

[Video example] (http://youtu.be/_KqGsnZOUA4)

Current behavior:

 - I saw next error:
![error](https://cloud.githubusercontent.com/assets/6644187/9074667/62fcf166-3ad1-11e5-8fc9-66013695f0e6.png)

- I saw next log:
```log
2015-08-04 22:41:22,995 1002 INFO share_bug openerp.modules.loading: 1 modules loaded in 15.23s, 0 queries
2015-08-04 22:41:23,124 1002 INFO share_bug openerp.modules.loading: updating modules list
2015-08-04 22:41:24,136 1002 INFO share_bug openerp.modules.loading: loading 23 modules...
2015-08-04 22:41:24,221 1002 INFO share_bug openerp.modules.loading: 23 modules loaded in 0.09s, 0 queries
2015-08-04 22:41:24,501 1002 INFO share_bug openerp.addons.base.ir.ir_model: Deleting 85@ir.actions.act_window (base.action_partner_mass_mail)
2015-08-04 22:41:24,620 1002 INFO share_bug openerp.modules.loading: Modules loaded.
```
 Notice a deleting of an act_window

Expected behavior:

 - Show the res.users form view done!

Last sha commit:
 - https://github.com/odoo/odoo/commit/4d9cef54125c8376f1a6dd50be395b1ade37dcad

Note:
 - If you uninstall the module `share` you don't have this problem anymore. (I think)

**Comment by beledouxdenis** [comment]
Hello Moises, 

This issue has been solved in the master branch by the following commit:
https://github.com/odoo/odoo/commit/cf63d4d277ef1ba02ff4ebcdae8583332a1775b1

You can have a look to the commit message for the full explanations.

Sadly, this issue cannot be solved in Odoo 8.0: We tried several other ways,which do not change the database structure (which is forbidden in stable releases, and the commit above does change the database structure), but it was simply not possible.

We hope you will understand.

Regards,

**Comment by moylop260** [comment]
I understand you
Thank you!
